mmv1/third_party/terraform/go.mod

```diff
@@ -38,6 +38,14 @@ require (
 	google.golang.org/protobuf v1.36.6
 )
 
+replace github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/hashicorp/terraform-plugin-sdk/v2 v2.37.0-beta.1
+
+replace github.com/hashicorp/terraform-plugin-go => github.com/hashicorp/terraform-plugin-go v0.27.0-alpha.1
+
+replace github.com/hashicorp/terraform-plugin-mux => github.com/hashicorp/terraform-plugin-mux v0.19.0-alpha.1
+
+replace github.com/hashicorp/terraform-plugin-testing => github.com/hashicorp/terraform-plugin-testing v1.13.0-beta.1
```

Comment:
I'm concerned about the idea of forcing these to point to alpha versions - unless this is just for testing?

Reply:
yeah this is just for initial testing, this would be changed once there's a non-alpha / non-beta offering

Reply:
i've removed the use of `terraform-plugin-testing -> 1.13.0-beta1` due to experiencing this [bug](https://github.com/hashicorp/terraform-plugin-testing/issues/327) when uprading to `1.6.0 >`